azerothcore/azerothcore-wotlk
# immolthar and mobs around are now immune to combat until the pylons are activated

## Changes Proposed:
- Add NOT_ATTACKABLE flag on creature create for highborne summoners and Immolthar
- Remove the above flag once the 5 pylons have been activated

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/8702

## SOURCE:


## Tests Performed:

- tested on win10, tried to rejoin a dungeon after I restarted the world and it seemed to stay then too
- Tested in normal run through dungeon and seems to work too.


## How to Test the Changes:


1. .go gameobject 29743, kill the mobs around it, the pylon de-activates
2. .go gameobject 29742, same
3. .go gameobject 29744, same
4. .go gameobject 29745, same.
5. Try to aggro immol'thar or the mobs around in any way. Should not be possible
6. .go gameobject 29746, kill mobs.
7. Immol'thar event should now start
8. all mobs should be attackable

try to reset, wipe, FD, whatever and see if it sticks properly.


## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
